### PR TITLE
Add mobilenet to benchmarks

### PR DESCRIPTION
**Pull request by jpienaar**
* Add mobilenet f32 and uint8 (randomly chosen between it and int8);
* Expand setup script to build more;
  - Needed to update a dir or two post local-task renaming;
  - I haven't rerun this script from scratch (was updated as I went
    along) as it builds quite a lot during but "it should work";
* Add noxnn variant for mobilebert to check hypothesis;

### CODE REVIEW

File: build_tools/benchmarks/comparisons/mobilenetv2_fp32_commands.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+from common.benchmark_command import *
+from common.benchmark_command_factory import BenchmarkCommandFactory
+
+_DEFAULT_NUM_BENCHMARK_RUNS = 50
+_DEFAULT_NUM_THREADS = 1
+
+
+class TfliteMobilenetV2FP32(TFLiteBenchmarkCommand):
+  """ Specializes the benchmark command to use TFLite. """
```
Comment (GMNGeoffrey) [inline build_tools/benchmarks/comparisons/mobilenetv2_fp32_commands.py:18]: Why are there spaces at the beginning of these? I don't think that's the style we use (https://google.github.io/styleguide/pyguide.html#383-functions-and-methods)

**Review by GMNGeoffrey — APPROVED** [review]
Can you specify in the description that this is the *manual* comparison benchmarks, not our automatically populated benchmark dashboard?

Please run it from scratch before merging. YOu can do it over lunch :-)

**Comment by jpienaar** [comment]
> Can you specify in the description that this is the _manual_ comparison benchmarks, not our automatically populated benchmark dashboard?
> 
> Please run it from scratch before merging. YOu can do it over lunch :-)

Hehe, I knew you would ask :)

**Comment by mariecwhite** [comment]
Looks good!

**Pull request merged**